### PR TITLE
API details from current window.location

### PR DESCRIPTION
**Pull request by rathorevaibhav**
## Summary 💬 

API details to be based on current frontend url

## Test Plan

Not needed

## How to test? - Please test thoroughly!!
1. Add any domain to your `/etc/hosts` file. For example `127.0.0.1   glific.test.com`
2. Verify that the backend app is accessible at http://glific.test.com:4000
3. Run the front-end server using the following command
```sh
> HOST=glific.test.com yarn start
```
4. Navigate to http://glific.test.com:3000

### CODE REVIEW

**Comment by rathorevaibhav** [comment]
@kurund @DigneshGujarathi I have a doubt on the `config/index.ts` file as it contains most of the things that are there in `common/constants.ts`. Let's discuss if there's scope for cleanup

**Review by kurund — CHANGES_REQUESTED** [review]
@rathorevaibhav Let's move all the URL related to config or add urls.ts and move all to that file.

Functionality is working as expected. Tested http and ws requests

**Comment by rathorevaibhav** [comment]
@kurund ready for review again

File: Dockerfile [hunk 1]
```diff
@@ -1,14 +1,10 @@
...
-ARG REACT_APP_GLIFIC_API
-ARG REACT_APP_WEB_SOCKET
-ARG REACT_APP_FLOW_EDITOR_API
+ARG REACT_APP_GLIFIC_API_PORT
 
 # Add those arg as env variable for builder
-ENV REACT_APP_GLIFIC_API $REACT_APP_GLIFIC_API
-ENV REACT_APP_WEB_SOCKET $REACT_APP_WEB_SOCKET
-ENV REACT_APP_FLOW_EDITOR_API $REACT_APP_FLOW_EDITOR_API
+ENV REACT_APP_GLIFIC_API_PORT $REACT_APP_GLIFIC_API_PORT
```
Comment (Satendra-SR) [inline Dockerfile:7]: 👍

File: README.md [hunk 2]
```diff
@@ -52,9 +52,7 @@ Your app is ready to be deployed!
 ## Docker image for production
 ```
 docker build \
---build-arg REACT_APP_GLIFIC_API=API-URL \
---build-arg REACT_APP_WEB_SOCKET=WEB-SOCKET-URL \
---build-arg REACT_APP_FLOW_EDITOR_API=FLOW-EDITOR-URL \
+--build-arg REACT_APP_GLIFIC_API_PORT=API-PORT \
```
Comment (Satendra-SR) [inline README.md:55]: Thanks for updating the readme as well 👍

**Review by kurund — APPROVED** [review]
Looks good!

**Pull request merged**